### ISSUE TITLE
"Received no partitions" on startup

### ISSUE THREAD

**Issue by pb30** [node-alarm-dot-com/homebridge-node-alarm-dot-com#110]
**Describe the bug**
Error in logs on startup. Devices eventually show up in Homebridge from Alarm.com, but looking at the code, the assumption that partitions always be set seems incorrect. System works fine without it. Not sure what `partitions` is referring to, security panel? Home is using Qualsys IQ Panel, so not a security system, just smart home panel.

```
[3/6/2023, 1:49:02 PM] [SmartHome] Logged into Alarm.com as xxx@gmail.com
[3/6/2023, 1:49:03 PM] [SmartHome] Error: Error: Received no partitions from Alarm.com
```

**To Reproduce**
Start homebridge

**Expected behavior**
No error

**ADC-SystemStates.json**
```
{
  id: xxxx,
  attributes: {
    description: "xxx Home",
    hasSnapShotCameras: false,
    supportsSecureArming: false,
    remainingImageQuota: 0,
    systemGroupName: '',
    unitId: xxx,
    accessControlCurrentSystemMode: 0,
    isInPartialLockdown: false,
    icon: 'property-single',
    canWaitForWifiValidation: false,
    canScanForWifi: true
  },
  partitions: [],
  sensors: [
    {
      id: 'xx-xx',
      type: 'devices/sensor',
      attributes: [Object],
      relationships: [Object]
    }
  ],
  lights: [
    {
      id: 'xx-xxx',
      type: 'devices/light',
      attributes: [Object],
      relationships: [Object]
    },
....
```

**Homebridge System (please complete the following information):**
* **Node.js Version**: 19.4.0
* **NPM Version**: 9.2.0
* **Homebridge Version**: 1.6.0>
* **Operating System**: macOS
* **Process Supervisor**: none

**Comment by chase9** [comment]
A partition is used in the security industry to separate parts of the system from each other with a single panel. For example, you could have a partition for your garage and have it operate independently from the rest of the house. I _thought_ that you always have at least a single partition which represents the common partition. 

Is [this what you have?](https://qolsys.com/iq-panel/) I'm curious about how you can have a panel but not a partition? It sounds like you're not able to arm your panel?

**Comment by pb30** [comment]
I think this is the model https://qolsys.com/iq-panel-4/

But yeah, no way to arm/disarm the panel. To the regret of the salesperson I just got the smart home package, not the security package. The panel I think basically just functions as Zwave hub for thermostat/deadbolt. Explains why the plug-in options for arm/disarm were confusing at first too. 

In quick local test replacing that throw error with continue seems fine. All accessories get loaded in on initial fetch instead of waiting for first timer refresh.

**Comment by pb30** [comment]
(Also, hey from another Minnesotan!)

**Comment by chase9** [comment]
Hey, hope you're all braced up for the snow tonight!

I had no idea this was possible, but it seems reasonable to not have that throw an error then. I'll switch it to debug.

**Comment by chase9** [comment]
Fixed in 999a4e4
